src/gm.datepickerMultiSelect.js

```diff
@@ -109,6 +111,8 @@ SOFTWARE.
 				return function (scope, element, attrs, ctrls) {
 					link.apply(this, arguments);
 
+					if(!angular.isDefined(scope.$parent.selectedDates)) return;
```

Comment:
Never mind- you're right. I think what you did is fine. Can you post an example in a Plunk? I really need to add unit tests for this.